docs/api-reference/next.config.js/url-imports.md

```diff
@@ -27,6 +27,12 @@ import { a, b, c } from 'https://example.com/modules/some/module.js'
 
 URL Imports can be used everywhere normal package imports can be used.
 
+## Security Model
+
+URL imports provide an opportunity for stronger security of sharing ES modules than what exists today.
```

Comment:
I’m not really sure what this means: “exists today”. Can we remove this sentence?

Reply:
Exists today with npm, we can drop it though 👍